crates/sui-tool/src/db_tool/mod.rs

```diff
@@ -15,19 +15,32 @@ pub enum DbToolCommand {
 }
 
 #[derive(Parser)]
+#[clap(rename_all = "kebab-case")]
 pub struct Dump {
-    /// If this is a gateway DB or authority DB
-    #[clap(long = "gateway")]
-    gateway: bool,
+    /// The type of store to dump
+    #[clap(long = "store")]
+    store_name: StoreName,
     /// The name of the table to dump
-    #[clap(long = "table_name")]
+    #[clap(long = "table-name")]
     table_name: String,
+    /// The size of page to dump. This is a u16
+    #[clap(long = "page-size")]
+    page_size: u16,
+    /// The page number to dump
+    #[clap(long = "page-num")]
+    page_number: usize,
 }
```

Comment:
do we need to update docs for DB tools?

Reply:
Good point. Will do as part of this issue
https://github.com/MystenLabs/sui/issues/3729